CHANGELOG.md

```diff
@@ -11,6 +11,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
  - Support to search multiple markets at once.
  - Support to search all available Spotify markets.
+ - Support to test whether the current user is following certain
+ users or artists
+ - Proper replacements for all deprecated playlist endpoints
+ (See https://developer.spotify.com/community/news/2018/06/12/changes-to-playlist-uris/ and below)
+
+### Deprecated
+
+- `user_playlist_change_details` in favor of `playlist_change_details`
+- `user_playlist_unfollow` in favor of `current_user_unfollow_playlist`
```

Comment:
If we have `playlist_is_following`, this could be `playlist_unfollow`?

Reply:
They have different functionality, however. `playlist_is_following` checks a list of given user ids (not defaulting to the current user) while `current_user_(un)follow_playlist` cannot take a list of ids. Do you still want to switch it?

Reply:
You are right, that makes sense